src/DynamoCore/PackageManager/MutatingFileCompressor.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Dynamo.PackageManager
+{
+    /// <summary>
+    ///     An IFileCompressor that actually attempts to compress a directory
```

Comment:
So it's you who has been putting these spaces in their doc comments? What's that all about?